src/providers/pdal/qgspdalprovider.h

```diff
@@ -66,6 +66,7 @@ class QgsPdalProvider: public QgsPointCloudDataProvider
     QVariantMap mOriginalMetadata;
     std::unique_ptr<QgsEptPointCloudIndex> mIndex;
     QgsPdalEptGenerationTask *mRunningIndexingTask = nullptr;
+    static QQueue<QgsPdalProvider *> mIndexingQueue;
```

Comment:
It is a common practice to have "s" prefix for static variables in QGIS code - could you please update that?